poll/poll.go

```diff
@@ -0,0 +1,140 @@
+package poll
```

Comment:
I'm not sure what this "poll" package is aimed to do, the name seems confusing to me, please can you add a package description?
If that's something specific for the bot, this should go inside the bot package. If its more generic, we need to modify all the code to use this new poll tipe.